lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -609,6 +610,27 @@ defmodule LivebookWeb.SessionLive.CellComponent do
   defp rounded_class(:top), do: "rounded-t-lg"
   defp rounded_class(:bottom), do: "rounded-b-lg"
 
+  defp doctest_summary(assigns) do
+    ~H"""
+    <div
+      :if={@doctest_summary.failures_count > 0}
+      class="flex flex-col pt-2 font-medium"
```

Comment:
I don't think we need any of these :)

```suggestion
      class="pt-2"
```